cli/src/main.rs

```diff
@@ -103,6 +116,9 @@ pub async fn main() {
         "appId": app_id,
         "alanVersion": format!("v{}", ALAN_VERSION),
       });
+      if env_file.is_some() {
+        body.as_object_mut().unwrap().insert(format!("envB64"), json!(get_env_file_b64(env_file.unwrap().to_string())));
```

Comment:
nit: should probably use `if let Some(env_file) = env_file {` to avoid unwrapping its contents